src/shallow-render.test.js

```diff
@@ -0,0 +1,9 @@
+const {h, Component} = require('preact');
+
+const {shallow} = require('./preact-render-spy');
+
+it('shallow renders the first level of Components', () => {
+  const Node = ({children}) => <div>{children}</div>;
+  const context = shallow(<Node><Node /></Node>);
+  expect(context.find('div').length).toBe(1);
```

Comment:
this expect is a bit confusing, but I get it...  it brings up some interesting alternate cases though.


* `shallow(<span><Node /></span>)` should still render the `Node` yeah?
* Should we add a test for `const ErrorIfRendered = () => throw new Error('should not ever render'), Node = () => <ErrorIfRendered />, context = shallow(<Node />)` to really ensure that the child component is never instantiated?